dspace-server-webapp/src/test/java/org/dspace/app/rest/matcher/CommunityMatcher.java

```diff
@@ -55,8 +53,6 @@ private CommunityMatcher() { }
     public static Matcher<? super Object> matchCommunityEntry(String name, UUID uuid, String handle) {
         return allOf(
             matchProperties(name, uuid, handle),
-            hasJsonPath("$._embedded.collections", Matchers.not(Matchers.empty())),
```

Comment:
again better overload the method than reduce the checks

Reply:
I agree here as well. I think we should *not* change the current behavior of `matchCommunityEntry()`.  Instead, we should create a new method called `matchCommunityEntryNoEmbeds()` which performs the match without embedded content.

Reply:
@abollini : This was resolved in a different manner. A new matchCommunityEntryFullProjection() method was added, and all existing tests that depend on embeds (full projection) were updated to use that new method.